tests/testthat/test-fbref.R

```diff
@@ -344,9 +344,9 @@ test_that("fb_player_goal_logs() works", {
 })
 
 Sys.sleep(3)
-test_that("fb_league_stats() works", {
+test_that("fb_league_stats() for players works", {
   testthat::skip_on_cran()
-
+  testthat::skip_on_ci()
```

Comment:
Skip the `fb_league_stats(..., team_or_player = "player")` tests on GitHub since they fail intermittently

Reply:
Good idea